src/DevL10N/xml/1.2/Languages/Japanese/Keyed/ToolsPawn.xml

```diff
@@ -1,76 +1,76 @@
+  <DebugAction_OffsetJoyNegative20>娯楽-20%</DebugAction_OffsetJoyNegative20>
+  <DebugAction_OffsetChemicalNegative20>薬物-20%</DebugAction_OffsetChemicalNegative20>
+  <DebugAction_SetSkill>スキルLv設定...</DebugAction_SetSkill>
+  <DebugAction_MaxSkill>スキルLv最大...</DebugAction_MaxSkill>
+  <DebugAction_MaxAllSkills>全スキルLv最大..</DebugAction_MaxAllSkills>
+  <DebugAction_MentalBreak>精神崩壊追加...</DebugAction_MentalBreak>
+  <DebugAction_MentalState>精神状態追加...</DebugAction_MentalState>
+  <DebugAction_StopMentalState>精神状態停止...</DebugAction_StopMentalState>
+  <DebugAction_Inspiration>やる気...</DebugAction_Inspiration>
+  <DebugAction_GiveTrait>特性を追加...</DebugAction_GiveTrait>
+  <DebugAction_SetBackstory>バックストーリーを変更...</DebugAction_SetBackstory>
+  <DebugAction_GiveAbility>追加 アビリティ...</DebugAction_GiveAbility>
+  <DebugAction_GivePsylink>追加 サイリンク...</DebugAction_GivePsylink>
+  <DebugAction_RemovePsychicEntropy>削除 ニューラルヒート</DebugAction_RemovePsychicEntropy>
+  <DebugAction_GiveGoodThought>追加 良い思考...</DebugAction_GiveGoodThought>
+  <DebugAction_GiveBadThought>追加 悪い思考...</DebugAction_GiveBadThought>
+  <DebugAction_ClearBoundUnfinishedThings>未完成品の作者除去</DebugAction_ClearBoundUnfinishedThings>
+  <DebugAction_ForceBirthday>強制 誕生日</DebugAction_ForceBirthday>
+  <DebugAction_Recruit>勧誘</DebugAction_Recruit>
+  <DebugAction_DamageApparel>衣服にダメージ</DebugAction_DamageApparel>
+  <DebugAction_WearApparel_ToSelected>選択者に衣服を付与...</DebugAction_WearApparel_ToSelected>
+  <DebugAction_EquipPrimary_ToSelected>選択者に武器を付与...</DebugAction_EquipPrimary_ToSelected>
+  <DebugAction_TameAnimal>動物を飼いならす</DebugAction_TameAnimal>
+  <DebugAction_TrainAnimal>動物を訓練する</DebugAction_TrainAnimal>
+  <DebugAction_TryDevelopBoundRelation>テスト 絆関係を構築する</DebugAction_TryDevelopBoundRelation>
+  <DebugAction_QueueTrainingDecay>訓練低下の予約</DebugAction_QueueTrainingDecay>
+  <DebugAction_StartMarriageCeremony>結婚式を開始...</DebugAction_StartMarriageCeremony>
+  <DebugAction_ForceInteraction>強制 社会関係</DebugAction_ForceInteraction>
+  <DebugAction_StartRandomGathering>開始 ランダムな集会</DebugAction_StartRandomGathering>
+  <DebugAction_StartGathering>開始 集会</DebugAction_StartGathering>
+  <DebugAction_StartPrisonBreak>開始 脱獄</DebugAction_StartPrisonBreak>
+  <DebugAction_PassToWorld>惑星を脱出</DebugAction_PassToWorld>
+  <DebugAction_Make1YearOlder>年齢を1歳上げる</DebugAction_Make1YearOlder>
+  <DebugAction_TryJobGiver>JobGiverを試みる</DebugAction_TryJobGiver>
+  <DebugAction_TryJoyGiver>JoyGiverを試みる</DebugAction_TryJoyGiver>
+  <DebugAction_EndCurrentJobInterruptForced>現在ジョブを中断 強制終了</DebugAction_EndCurrentJobInterruptForced>
+  <DebugAction_CheckForJobOverride>ジョブの上書きを確認</DebugAction_CheckForJobOverride>
+  <DebugAction_ToggleJobLogging>切替 ジョブの記録</DebugAction_ToggleJobLogging>
+  <DebugAction_ToggleStanceLogging>切替 stanceの記録</DebugAction_ToggleStanceLogging>
+  <DebugAction_Kidnap>入植者誘拐</DebugAction_Kidnap>
+  <DebugAction_Selected_SetFacing>作成 選択 face cell</DebugAction_Selected_SetFacing>

```

Comment:
SetFacing can be used to make your colonist _face_ north, east, south, or west direction. Thus, in my opinion, it can be translated. Please confirm this.

Reply:
Yes. I also understand the meaning of the tool, but I could not translate it well. Let's think about a better translation.